### ISSUE TITLE
Make sure that used images conform to requirements

### ISSUE THREAD

**Issue by lhstrh** [lf-lang/lingua-franca#832]
See https://wiki.eclipse.org/Platform-releng/Updating_Branding

 - [ ] Check Epoch images (https://wiki.eclipse.org/Platform-releng/Updating_Branding)
 ```
[IconExe] Error - 7 original icon(s) not replaced in /tmp/p2.brandingIron8424790835348554983/launcher.exe:
	- 48x48, 8 bits, 3752 byte(s) @ 121872
	- 32x32, 8 bits, 2216 byte(s) @ 125624
	- 16x16, 8 bits, 1384 byte(s) @ 127840
	- 48x48, 32 bits, 9640 byte(s) @ 129224
	- 32x32, 32 bits, 4264 byte(s) @ 138864
	- 16x16, 32 bits, 1128 byte(s) @ 143128
	- 256x256, 32 bits, 270376 byte(s) @ 144256
[IconExe] For more details, see https://git.eclipse.org/c/equinox/rt.equinox.p2.git/tree/bundles/org.eclipse.equinox.p2.publisher.eclipse/src/org/eclipse/pde/internal/swt/tools/IconExe.java
```

**Comment by lhstrh** [comment]
I've updated the splash screen so that it hopefully does not distort as on MacOS anymore. Whatever other incompatibilities there might be, I'm not sure. The [guidelines with respect to icons](https://wiki.eclipse.org/Platform-releng/Updating_Branding) are rather unclear and, frankly, ridiculous. If someone else wants to volunteer to pick up this task, feel welcome, but I'm going to just leave this as is.

**Comment by lhstrh** [comment]
Looks like [my fix for the splash screen](7a2e9a6ab71ef65128984daa69f37972c37216d7) also didn't work (the splash is still rendered pink on MacOS). I followed the instructions for producing the image; I'm not sure what else to do. @a-sr: do you have any suggestions?
